File: ST_lab/pro1.c
#include<stdio.h>
#include<stdlib.h>
#include<ctype.h>
#include<conio.h>

int main()
{
  int a, b, c;
  clrscr();
  printf("Enter three sides of the triangle : ");
  scanf("%d %d %d", &a, &b, &c);

  if((a > 10) || (b > 10) || (c > 10)){
    printf("Out of range");
    getch();
    exit(0);
  }
  if((a<b+c)&&(b a+c)&&(c<a+b)){
    if((a==b)&&(b==c)){
      printf("Equilateral triangle");
    }
    else if((a!=b)&&(a!=c)&&(b!=c)){
      printf("Scalene triangle");
    }
  else
    printf("Isosceles triangle");
  }
  else {
    printf("triangle cannot be formed");
  }
  getch();
  return 0;
}

File: ST_lab/pro1.py
"""
Program 1:

Design and develop a program in a language of your choice to solve the triangle problem
defined as follows : Accept three integers which are supposed to be the three sides of
triangle and determine if the three values represent an equilateral triangle, isosceles
triangle, scalene triangle, or they do not form a triangle at all. Derive test cases for your
program based on decision-table approach, execute the test cases and discuss the results

"""

print("Enter 3 integers which are sides of triangle")

a = int(input("Enter A : "))
b = int(input("Enter B : "))
c = int(input("Enter C : "))

print(f"A = {a}\t , B = {b}\t, C = {c}")

# to check is it a triangle or not

if( a < b+c and b < a+c and c < a+b ):
    istriangle ='y'
else:
    istriangle ='n'

if (istriangle=='y'):
    if ((a==b) and (b==c)):
        print("equilateral triangle\n")
    elif ((a!=b) and (a!=c) and (b!=c)):
        print("scalene triangle\n")
    else:
        print("isosceles triangle\n")
else:
    print("Not a triangle\n")
